apps/src/codebridge/InfoPanel/ValidatedInstructions.tsx

```diff
@@ -335,6 +335,7 @@ const ValidatedInstructions: React.FunctionComponent<InstructionsProps> = ({
   return (
     <div
       id="instructions"
+      data-theme="Light"
```

Comment:
Does this still need `data-theme` if it's set by the ThemeProvider wrapper?

Reply:
We don't want this in Codebridge since we use the dark theme for everything (we set it [here](https://github.com/code-dot-org/code-dot-org/blob/46a7f46d4f80387b01093d5ba362a8cc198753f0/apps/src/pythonlab/PythonlabView.tsx#L199) for Python Lab and [here](https://github.com/code-dot-org/code-dot-org/blob/46a7f46d4f80387b01093d5ba362a8cc198753f0/apps/src/weblab2/Weblab2View.tsx#L122) for Web Lab 2, which we may be able to remove now?)

Reply:
Thanks @molly-moen, I was wondering exactly this.  I've removed it from here, and from those two places as well.